bokehjs/src/lib/core/enums.ts

```diff
@@ -62,6 +62,8 @@ export const FlowMode = Enum("block", "inline")
 export type FontStyle = "normal" | "italic" | "bold" | "bold italic"
 export const FontStyle = Enum("normal", "italic", "bold", "bold italic")
 
+export type FontStyleSpecValue = {value: FontStyle} | {field: string}
```

Comment:
I'm not sold on any of these names btw, open to ideas

Reply:
Usually we'd go with `FontStyleSpec` and similar to `ColorSpec` this should already exist:

https://github.com/bokeh/bokeh/blob/branch-3.4/bokehjs/src/lib/core/properties.ts#L739

So I think the next task is to figure out how to leverage the existing dataspecs (or understand why they are not sufficient)

Reply:
Done in last commit